src/components/TodoForm.vue

```diff
@@ -105,13 +110,16 @@ function toggleEdit(toDo: Todotype) {
   toDo.isEditing = !toDo.isEditing;
 }
 function addTodo() {
-  todos.value.push({
+  const date = moment().format('DD.MM.YYYY');
+  todos.value.unshift({
     text: 'Add your Todo',
     isEditing: false,
     priorityChange: false,
```

Comment:
Please update the priorityChange attribute name to isPriorityChanged in order to use the `is` prefix, because this is a boolean variable.